src/openrct2/localisation/Localisation.cpp

```diff
@@ -1512,8 +1512,9 @@ money32 string_to_money(const char* string_to_monetise)
 
     auto number = std::stod(processedString, nullptr);
     number /= (currencyDesc->rate / 10.0);
-    auto whole = static_cast<uint16_t>(number);
-    auto fraction = static_cast<uint8_t>((number - whole) * 100);
```

Comment:
What you will likely work, but it's not really the _right_ way to fix it. It's something I'd only want to do if there is no other way.

What happens if you change `100` to `100.0`? And if that alone doesn't work, what happens if you do `static_cast<uint8_t>(ceil((number - whole) * 100.0))`?

Reply:
In my testing, none of the expected solutions worked. I tried changing to 100.0, using ceiling, floor, rounding functions, etc. I'm not sure if it could have been a problem with my machine, but every fix resulted in a floating point error unless I performed the subtraction before assigning to variable.

Reply:
I have done some tests myself, and found a way to do it using `ceil`. I'll therefore close this PR, but I might reopen it should my solution not be universal.